Antidote/ActiveCallViewController.m

```diff
@@ -378,6 +415,8 @@ - (void)updateTimerLabel
     self.subLabel.text = [NSString stringFromTimeInterval:self.callDuration];
 
     [self.subLabel setNeedsDisplay];
+
+    [self removeFriendAvatar];
```

Comment:
I think it would be better to rename method to `hideCallPausedByFriendIfNeeded`.